letsencrypt-win-simple/Program.cs

```diff
@@ -80,6 +81,27 @@ static void Main(string[] args)
             Log.Information("Config Folder: {configPath}", configPath);
             Directory.CreateDirectory(configPath);
 
+            certificatePath = Properties.Settings.Default.CertificatePath;
+
+            if (string.IsNullOrWhiteSpace(certificatePath))
+            {
+                certificatePath = configPath;
+            }
+
+            try
+            {
+                Directory.CreateDirectory(certificatePath);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Error creating the certificate directory: " + certificatePath);
+                Log.Error("Error creating the certificate directory. Error: {@ex}", ex);
```

Comment:
Change the Log.Error to a Log.Warning and say it was defaulted to the config path.